api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -399,7 +399,26 @@ public void saveConceptSource_shouldNotSetDateCreatedIfOneIsSuppliedAlready() th
 		
 		Assert.assertEquals(newConceptSource.getDateCreated(), expectedDate);
 	}
-	
+
+	/**
+	 * @see {@link ConceptService#saveDrug(Drug)}
+	 */
+	@Test
+	@Verifies(value = "should pass when drug index is updated on save", method = "saveDrug(Drug)")
+	public void saveDrug_shouldUpdateDrugIndex() throws Exception {
+		Drug drug = new Drug();
+		drug.setName("ASPIRIN BRANDED");
+		drug.setConcept(conceptService.getConceptByName("ASPIRIN"));
+		drug.setStrength("500mg");
+
+		Drug saveDrug = conceptService.saveDrug(drug);
+		assertNotNull(saveDrug);
+
+		List<Drug> drugs = conceptService.getDrugs("ASPIRIN BRANDED");
+		assertEquals(1, drugs.size());
+		assertEquals("ASPIRIN BRANDED", drugs.get(0).getName());
```

Comment:
@rkorytkowski : Thanks for the pointer. I did not knew search index is updated after every transaction. Closing this pull request as not required.